parsers/rhea/sql.go

```diff
@@ -0,0 +1,182 @@
+package rhea
+
+import (
+	"database/sql"
+	"errors"
+)
+
+/******************************************************************************
+
+Database functions
+
+These functions take in the rhea.rdf.gz dump file and import them into an sqlite
+database. This database can be used locally for mapping functions, but more importantly,
+it is used because it can enforce the relationships between each part of Rhea and alert
+the user if the parser is failing to pick up on any of the relationships in the Rhea
+database.
+
+******************************************************************************/
+
+// RheaSqlite inserts the Rhea database into an SQLite database with proper normalization for advanced queries.
+func RheaSqlite(db *sql.DB, rhea Rhea) error {
+	// Start transaction with database for insertion. This ensures if there are any problems, they are seamlessly rolled back
+	tx, err := db.Begin()
+	if err != nil {
+		return err
+	}
+
+	schema := `
+	-- Rhea tables --
```

Comment:
Since this schema is so complicated, you may want to document it with: https://dbdiagram.io/home and add it to the PR.